modules/ROOT/pages/ogm/subscriptions.adoc

```diff
@@ -0,0 +1,244 @@
+[[ogm-subscriptions]]
+= How to use the OGM with subscriptions
+
+The Neo4j GraphQL Library can be used in conjunction with the OGM in order to extend the Library's functionalities, or to take advantage of the xref:ogm/private.adoc[`@private` directive]. 
+
+== Usage
+
+This section shows how to use subscriptions with the OGM inside custom resolvers.
+
+=== Prerequisites
+
+. Use the following type definitions:
+[source, javascript, indent=0]
+----
+const typeDefs = `#graphql
+    type User {
+      email: String!
+      password: String! @private
+    }
+`;
+----
+
+. Set up a subscriptions plugin instance to be shared between the Library and the OGM constructors:
+[source, javascript, indent=0]
+----
+const subscriptionsPlugin = new Neo4jGraphQLSubscriptionsSingleInstancePlugin();
+----
+
+. Set up a server that supports subscriptions according to the xref:subscriptions/getting-started.adoc#setting-up-server[subscriptions reference usage].
+
+
+=== Adding the OGM
+
+In order to utilize the `@private` marked field on the `User` type, you need to create the `User` model.
```

Comment:
I wonder if this line shouldn't be a step in the guide with an example of how to do that?